### PR TITLE
Reworks Adrenaline Glands into Repurposed Glands (Realingment)

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Adrenaline Glands have been removed and replaced with Repurposed Glands.

This ability removes use of your arms in exchange for regenerating your legs, breaking cuffs, giving you speed, damage slowdown immunity, and regenerating and doubling your stamina temporarily.

It's probably a bit overtuned right now. I remembered how useless this ability is and it made me mad.

## Why It's Good For The Game

I'm tired of AG being a completely useless noob trap that doesn't do anything of value and isn't even accurate to its description or flavor. Taking inspiration from Realignment, this turns it into what skoglol tried and failed to turn it into: An escape plan that can't be used as free stimulants.

## Changelog

:cl:
balance: Reworks Adrenaline Glands into Repurposed Glands (Realingment). Lose the use of your arms, but you can escape any situation!
/:cl:

### CODE REVIEW

File: code/modules/antagonists/changeling/powers/adrenaline.dm [hunk 1]
```diff
@@ -1,19 +1,54 @@
...
+
+	// Get us standing up.
+	user.SetAllImmobility(0)
+	user.setStaminaLoss(0)
+	user.set_resting(FALSE, instant = TRUE)
+
+	// Add fast reagents to go fast.
 	user.reagents.add_reagent(/datum/reagent/medicine/changelingadrenaline, 4) //20 seconds
-	user.reagents.add_reagent(/datum/reagent/medicine/changelinghaste, 3) //6 seconds, for a really quick burst of speed
+	user.reagents.add_reagent(/datum/reagent/medicine/changelinghaste, 10) //20 seconds
```
Comment (carlarctg) [inline code/modules/antagonists/changeling/powers/adrenaline.dm:46]:
```suggestion
```

this one is ovrkull they can just use strained muscles

**Comment by moocowswag** [comment]
I dont even think its adrenal's fault, stamina regeneration is useless outside of vast amounts because you need to overcome the extra magic 30 you take when you enter stam crit (making a single baton hit deal 90 stamina damage, any if you regenerate out of stam crit via stamina regen the next baton hit will get that extra 30 again)

**Comment by improvedname** [comment]
in the long run though do we need to remove adrenal glands cant we just have both i think lings being able to conjure ephedrine is still kinda neat from a offensive capability

**Comment by carlarctg** [comment]
strained muscles does the same thing better

**Comment by ATH1909** [comment]
now we need the inverse power where you skip leg day for increased arm strength

wheelchair changeling new meta

File: code/modules/antagonists/changeling/powers/adrenaline.dm [hunk 2]
```diff
@@ -1,19 +1,53 @@
...
+		qdel(O)
+		user.visible_message(span_warning("[user]'s legs suddenly rip [O] apart!"), \
+			span_warning("We rip apart our leg restraints!"))
+
+	// Regenerate our legs only.
+	var/our_leg_zones = (GLOB.all_body_zones - GLOB.leg_zones)
+	user.regenerate_limbs(excluded_zones = our_leg_zones) // why is this exclusive rather than inclusive
+
+	// Upgrade our stamina, lose our arms!
+	user.max_stamina *= 2
```
Comment (Jacquerel) [inline code/modules/antagonists/changeling/powers/adrenaline.dm:33]: Did you test this and does it work?
IIRC stamcrit is based on maxhealth and not max_stamina.

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/changeling/powers/adrenaline.dm (hunk 2 above)
Comment (carlarctg) [reply code/modules/antagonists/changeling/powers/adrenaline.dm:33]: fuckin wot

Comment (carlarctg) [reply code/modules/antagonists/changeling/powers/adrenaline.dm:33]: image

??????????????????????????????? ok

**Comment by carlarctg** [comment]
Since stamina is STUPID i made it so that chagnelige adrenaline now reovers 30 stamina per tick. Thats the onyl place its used so it'll be fine since they got no arms

**Comment by Metekillot** [comment]
Why is it bad for changeling adrenaline to increase your combat effectiveness? You said "free stimulants" as if stimulants are even really any good against stamcrit.

**Comment by Metekillot** [comment]
Also #83556 would conflict with this

**Comment by carlarctg** [comment]
> Why is it bad for changeling adrenaline to increase your combat effectiveness? You said "free stimulants" as if stimulants are even really any good against stamcrit. And even if it did match stimulants totally... It's not free you have to pay chemicals for it

Not sure how you missed the point of the PR but this is meant to make the ability be usable as an escape, like it used to be 

I have no opinions on stims beyond that

**Pull request merged**